pages/services/dse/3.0.0-6.7.2/security/index.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: layout.pug
+navigationTitle: Security for DSE
+excerpt: Security, encryption and authorization for DSE
+title: Security
+menuWeight: 50
+model: /services/dse/data.yml
+render: mustache
+---
+
+# DC/OS {{ model.techName }} Security
+
+The DC/OS {{ model.techName }} service supports {{ model.techShortName }}'s native transport encryption mechanisms. The service provides automation and orchestration to simplify the usage of these important features.
+
+<p class="message--note"><strong>NOTE: </strong> These security features are only available on DC/OS Enterprise 1.10 and later. </p>
+
+## Transport Encryption
+
+#include /services/include/security-transport-encryption-lead-in.tmpl
+
+#include /services/include/security-configure-transport-encryption.tmpl
+
+#include /services/include/security-transport-encryption-clients.tmpl
+
+## DSE Authentication/Authorization Schemes
+{{ model.shortTechName }} in DC/OS currently supports both internal and ldap authentication/authorization schemes.  You can configure both schemes and then select the order in which they are used, or you can configure just one, in which case only that scheme will be used.  {{ model.shortTechName }} will try to authenticate with the default scheme first and fall back to the alternate scheme if it has been configured. More information about how {{ model.shortTechName }} handles this can be found in [DataStax's documentation](http://docs.datastax.com/en/dse/5.1/dse-admin/datastax_enterprise/security/secDSEUnifiedAuthAbout.html)
+
+## Enabling {{ model.shortTechName }} Authentication/Authorization
+To enable {{ model.shortTechName }} authentication/authorization, follow these steps:
+   1. In Advanced Installation wizard, configure the following fields:
+      ```
+      In "cassandra" tab,
```

Comment:
Why are these instructions formatted as CODE BLOCKS? Why would users want to cut and paste instructions?

Reply:
This is a remnant of the previous versions but TBH I think it's easier on the eyes that way. See latest public docs: https://docs.mesosphere.com/services/dse/2.4.0-5.1.10/security/#ldap-configuration

If every field were its own bullet/ordered list the instructions would look cumbersome. I can change it but maybe it's better to separate that chunk of work and do it across the previous 2 (or more) versions

Reply:
OK. I'll leave it as is. Thanks.